extensions/wikia/PhalanxII/PhalanxStatsSpecialController.class.php

```diff
@@ -42,55 +42,38 @@ public function index() {
 	}
 
 	private function blockStats($blockId) {
-		$this->wg->Out->setPageTitle( sprintf( "%s #%s", wfMsg('phalanx-stats-title'), $blockId ) );
-		$this->wg->Out->addBacklinkSubtitle( $this->title );
+		$out = $this->getOutput();
+		$out->setPageTitle( $this->msg( 'phalanx-stats-title' )->text() . "#$blockId" );
+		$out->addBacklinkSubtitle( $this->title );
 
 		$data = Phalanx::newFromId($blockId);
 
 		if ( !isset( $data["id"] ) ) {
-			$this->wg->Out->addWikiMsg( 'phalanx-stats-block-notfound', $blockId );
-			return;
+			$out->addWikiMsg( 'phalanx-stats-block-notfound', $blockId );
+			return false; // skip rendering
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)